makemkv/makemkv.py

```diff
@@ -246,17 +246,16 @@ def _translate_codes(
 
         return key, return_value
 
-    def _run(self, cmd: list[str]) -> MakeMKVOutput:
-        """Run makemkvcon and parse its output."""
-        p = self.process = Popen(cmd, stderr=STDOUT, stdout=PIPE, bufsize=1, text=True)
-        logger.info('Running "%s"', " ".join(cmd))
+    def _parse_makemkv(self, lines, p=None) -> dict:
```

Comment:
`_parse_makemkv_log` would be a more precise name.

Could you please provide type annotations? I think `lines` should be `Iterable[str]` (with `Iterable` being imported from `typing`). Return type should be `MakeMKVOutput`.

`p` can be ommited and all references should be replaced with `self.process`.

So the overall method signature would look like
```python
def _parse_makemkv_log(self, lines: Iterable[str]) -> MakeMKVOutput:
```